cmd/cluster-kube-apiserver-operator/render/render.go

```diff
@@ -32,65 +53,137 @@ func NewRenderCommand() *cobra.Command {
 			}
 		},
 	}
-	cmd.Flags().StringVar(&renderOpts.namespace, "namespace", "openshift-kube-apiserver", "Target namespace.")
-	cmd.Flags().StringVar(&renderOpts.image, "image", "openshift/origin-hypershift:latest", "Image to use.")
 
-	cmd.Flags().StringVar(&renderOpts.assetInputDir, "asset-output-dir", "", "Output path for rendered assets.")
-	cmd.Flags().StringVar(&renderOpts.assetOutputDir, "asset-input-dir", "", "A path to directory with certificates and secrets.")
-	cmd.Flags().StringVar(&renderOpts.manifestTemplatesDir, "manifest-templates-dir", "/usr/share/bootkube/manifests", "A path to directory with manifest templates.")
+	cmd.Flags().StringVar(&renderOpts.manifest.namespace, "manifest-namespace", "openshift-kube-apiserver", "Target namespace for API server pods.")
+	cmd.Flags().StringVar(&renderOpts.manifest.image, "manifest-image", "openshift/origin-hypershift:latest", "Image to use for the API server.")
```

Comment:
this is not really a 'manifest-image' right? it is the api server image to use... i guess the pattern here to be the manifest struct fields have manifest prefix :) </bike_shedding>

Reply:
🍿